tests/modules/antismash/antismashlite/main.nf

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ANTISMASH_ANTISMASHLITE } from '../../../../modules/antismash/antismashlite/main.nf'
+include { ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES } from '../../../modules/antismash/antismashlitedownloaddatabases/main.nf'
+include { GUNZIP as GUNZIP1 } from '../../../../modules/gunzip/main.nf'
+include { GUNZIP as GUNZIP2 } from '../../../../modules/gunzip/main.nf'
+include { UNTAR as UNTAR1 } from '../../../../modules/untar/main.nf'
+include { UNTAR as UNTAR2 } from '../../../../modules/untar/main.nf'
+include { UNTAR as UNTAR3 } from '../../../../modules/untar/main.nf'
+
+workflow test_antismashlite {
+    genome_fna = [
+        [ id:'test' ],
+        file(params.test_data['bacteroides_fragilis']['genome']['genome_fna_gz'], checkIfExists: true)
+    ]
+
+    genome_gff = [
+        [],
+        file(params.test_data['bacteroides_fragilis']['genome']['genome_gff_gz'], checkIfExists: true)
+    ]
+
+    antismash_db1 = [
```

Comment:
I would call each db channel name e.g. `antismash_css`, `antismash_detection` etc for clarity, these aren't different databses, just different components of the same one